testsuite/smoke-test/src/rest_api.rs

```diff
@@ -51,3 +59,233 @@ async fn test_basic_client() {
 
     info.client().get_transactions(None, None).await.unwrap();
 }
+
+#[tokio::test]
+async fn test_bcs() {
```

Comment:
what's the advantage of this over the REST api tests?

Reply:
The advantage here is I properly tested the REST client, which we had no tests for.

It's actually the reason we missed version -> ledger_version